test/moneyLender.js

```diff
@@ -0,0 +1,153 @@
+        .then(() => debts.BorrowMoney(value, {from: BORROWER}))
+        .then(() => debts.debt(BORROWER))
+            then(asserts.equal(value)); 
+  });
+
+  it('should emit Borrow event on borrow', () => {
+    const value = 1000;
+    return Promise.resolve()
+    .then(() => debts.BorrowMoney(value, {from: BORROWER}))
+    .then(result => {
+      assert.equal(result.logs.length, 1);
+      assert.equal(result.logs[0].event, 'Borrowed');
+      assert.equal(result.logs[0].args.from, BORROWER);
+      assert.equal(result.logs[0].args.amount.valueOf(), value);
+    });
+  });
+
+  it('should allow to repay', () => {
+    const value = 1000;
+    return Promise.resolve()
+    .then(() => debts.BorrowMoney(value, {from: BORROWER}))
+    .then(() => debts.MoneyBack(BORROWER, value, {from: BANK}))
+    .then(() => debts.debt(BORROWER))
+      .then(asserts.equal(0));
+  });
+
+  it('should emit Back event on money back', () => {
+    const value = 1000;
+    return Promise.resolve()
+      .then(() => debts.BorrowMoney(value, {from: BORROWER}))
+      .then(() => debts.MoneyBack(BORROWER, value, {from: BANK}))
+        .then(result => {
+        assert.equal(result.logs.length, 1);
+        assert.equal(result.logs[0].event, 'Backed');
+        assert.equal(result.logs[0].args.from, BORROWER);
+        assert.equal(result.logs[0].args.amount.valueOf(), value);
+      });
+
+  });
+
+  it('should allow to check am I able to borrow', () => {
+    const value = 1000;
+    return Promise.resolve()
+    .then(() => debts.BorrowMoney(value, {from: BORROWER}))
+    .then(() => debts.isAbleToGetMoney({from: BORROWER}))
+      .then(asserts.equal(false))
+    });
+
+  it('should be 0 debt by default', () => {
+     return Promise.resolve()
+    .then(() => debts.debt(BORROWER))
+      .then(asserts.equal(0));
+  });
+
+  it('should allow to get a money by default', () => {
+     return Promise.resolve()
+    .then(() => debts.isAbleToGetMoney({from: BORROWER}))
+      .then(asserts.equal(true))
+  });
+
+  it('should allow to borrow from several accounts', () => {
+  const secondaryBorrower = accounts[3];
+  const value = 1000;
+  return Promise.resolve()
+    .then(() => debts.BorrowMoney(value, {from: BORROWER}))
+    .then(() => debts.BorrowMoney(value, {from: secondaryBorrower}))
+    .then(() => debts.debt(BORROWER))
+      .then(asserts.equal(value))
+    .then(() => debts.debt(secondaryBorrower))
+      .then(asserts.equal(value));
+  });
+
+  it('should not allow owner to borrow', () => {
+    const value = 1000;
+    return Promise.resolve()
+    .then(() => debts.BorrowMoney(value, {from: BANK}))
+    .then(() => debts.debt(BANK))
+      .then(asserts.equal(0)); 
+  });
+
+  it('should not allow to repay from owner wallet', () => {
+    const value = 1000;
+    return Promise.resolve()
+      .then(() => debts.BorrowMoney(value, {from: BORROWER}))
+      .then(() => debts.MoneyBack.call(BANK, value, {from: BANK}))
```

Comment:
Just make sure you understand that `.call` can never change contract's state, so it is usually pointless to do the check on line 109.